### ISSUE TITLE
reword one of examples for role presentation

### ISSUE THREAD

**Issue by jnurthen** [w3c/aria#1455]
The example states:
"An image that is in a container with the img role and where the full text alternative is available and is marked up with aria-labelledby and (if needed) aria-describedby;"

This took me multiple times to parse what was meant - needs rewording to avoid confusion.

My reading is that it is trying to describe the following
```
<div role="img" aria-labelledby="a">
  <span id="a">Alternative text for the image</span>
  <img role="presentation">
</div>
```

**Comment by aleventhal** [comment]
Aren't children of an img automatically presentational?

**Comment by jnurthen** [comment]
@aleventhal yes but I'd still expect a validator to 🤮 if it comes across an IMG with no alt or role presentation even as a child of a role=img

**Comment by jnurthen** [comment]
but yeah - not sure the explicit  adds anything more than confusion
